test/test_mps.py

```diff
@@ -10332,30 +10385,18 @@ def req_grad(t):
 
                 self.assertEqual(cpu_grad_inputs, mps_grad_inputs, atol=atol, rtol=rtol)
             except Exception as e:
-                if not generate_new_truth:
-                    raise e
+                raise e
```

Comment:
Is the variable below used? If not, we can remove this try/except as well!

Reply:
Done